velox/substrait/SubstraitToVeloxExpr.cpp

```diff
@@ -73,6 +95,10 @@ SubstraitVeloxExprConverter::toVeloxExpr(
     case ::substrait::Expression_Literal::LiteralTypeCase::kBoolean:
       return std::make_shared<core::ConstantTypedExpr>(
           variant(substraitLit.boolean()));
+    case ::substrait::Expression_Literal::LiteralTypeCase::kI16:
+      // substraitLit.i16() will return int32, So we need this type conversion.
```

Comment:
Could you change this comment a little? Captalize the first letter and change `So` to be lowercase.